### PR TITLE
892777-.NET MAUI UG corrections

### PR DESCRIPTION
**Pull request by SivaranjithN**
Hi Team,
I have updated the Getting-Started page of .NET MAUI SfAutocomplete control .

Regards,
Sivaranjith N.

### CODE REVIEW

File: MAUI/Autocomplete/Getting-Started.md [hunk 1]
```diff
@@ -55,13 +55,11 @@ namespace AutocompleteSample
 
 ## Create a Simple .NET MAUI Autocomplete
 
-The .NET MAUI Autocomplete control is configured entirely in C# code or by using XAML markup. The following steps explain how to create a [.NET MAUI Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) (SfAutocomplete) and configure its elements:
+The .NET MAUI Autocomplete control can be configured entirely using C# code or XAML markup.. The following steps explain how to create a [.NET MAUI Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) (SfAutocomplete) and configure its elements:
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Getting-Started.md:58]: There are two periods. remove one.

Comment (SivaranjithN) [reply MAUI/Autocomplete/Getting-Started.md]: Addressed the comment.

File: maui-toc.html [hunk 2]
```diff
@@ -164,6 +164,7 @@
         <ul>
             <li><a href="/maui/Autocomplete/Overview">Overview</a></li>
             <li><a href="/maui/Autocomplete/Getting-Started">Getting Started</a></li>
+			<li><a href="/maui/Autocomplete/Basic Features">Basic Features</a></li>
```
Comment (PrithisIyyappan) [inline maui-toc.html:167]: Change it to Basic-Features

Comment (PrithisIyyappan) [reply maui-toc.html:167]: in path.

File: MAUI/Autocomplete/Basic Features.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Basic Features with .NET MAUI Autocomplete | Syncfusion
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Basic Features.md:3]: check this.

File: MAUI/Autocomplete/Basic Features.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+![.NET MAUI Autocomplete with single selection mode](Images/GettingStarted/SingleSelection.gif)
+
+You can access the full getting started sample for .NET MAUI Autocomplete by following this [link](https://github.com/SyncfusionExamples/maui-autocomplete-samples).
+
+## Text
+
+The [Text](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.DropDownControls.DropDownListBase.html#Syncfusion_Maui_Inputs_DropDownControls_DropDownListBase_Text) property is used to get the user-submitted text in the [SfAutoComplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html). The default value of the `Text` property is `string.Empty`.
+
+N> You can refer to our [.NET MAUI Autocomplete](https://www.syncfusion.com/maui-controls/maui-autocomplete) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Autocomplete Example](https://github.com/SyncfusionExamples/maui-autocomplete-samples) that shows you how to render the Autocomplete in .NET MAUI.
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Basic Features.md:44]: Is this note needed here or on the Getting Started page?

File: maui-toc.html (hunk 2 above)
Comment (SivaranjithN) [reply maui-toc.html]: Changed as mentioned.

File: MAUI/Autocomplete/Basic Features.md (hunk 3 above)
Comment (SivaranjithN) [reply MAUI/Autocomplete/Basic Features.md]: Checked and updated.

File: MAUI/Autocomplete/Basic Features.md (hunk 4 above)
Comment (SivaranjithN) [reply MAUI/Autocomplete/Basic Features.md]: It is needed in Getting-Started page so moved there.

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**